odoo_conn.php

```diff
@@ -1,36 +1,38 @@
 <?php
-/*
-Plugin Name: Contact 7 to Odoo connector
-Plugin URI: https://www.jackdane.co.uk
-Description: Connect your WordPress Contact 7 Forms to Odoo
-Version: 0.1.2
-Requires PHP: 7.3
-Author: Jack Dane
-Author URI: https://www.jackdane.co.uk
-*/
-
-add_option("odoo_conn_db_version");
-
-require("vendor/autoload.php");
-
-require("encryption.php");
-
-require("odoo_connector/odoo_connector.php");
-
-require("dependency_check.php");
-
-register_activation_hook(__FILE__, "\\odoo_conn\\dependency_check\\odoo_conn_contact_7_plugin_active");
-
-require("activation.php");
-
-register_activation_hook(__FILE__, "odoo_conn_activation_function");
-
-require("deactivation.php");
-
-register_deactivation_hook(__FILE__, "odoo_conn_deactivation_function");
-
-require("loaded.php");
-
-require("admin/main.php");
-
-?>
\ No newline at end of file
+/**
+ * Plugin Name: Contact Form 7 to Odoo connector
+ * Description: When submitting a form through Contact Form 7, the data is extracted and mapped to the designated fields in Odoo.
+ * Author: Jack Dane
+ * Author URI: https://www.jackdane.co.uk
+ * Plugin URI: https://github.com/Jack-Dane/odoo-wp-plugin
+ * Requires PHP: 7.3
+ * Tested up to: 6.5.3
```

Comment:
Is this option currently supported? 

I can't see it mentioned in the documentation: https://developer.wordpress.org/plugins/plugin-basics/header-requirements/

And there was a feature request for it (that doesn't look to be completed): https://core.trac.wordpress.org/ticket/51139

But I can see the proposed changes live when viewing plugins on the store. Do you know if there is any official documentation about this option?

Reply:
You are right.

"Tested up to: " it's usually included in the Readme Header Information (readme.txt).
(https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#readme-header-information)